[Liam] Clarify multiple data types and add info about metadata order (#203)
VerifiedInc/VerifiedInc.github.io: versioned_docs/version-2/1-click-signup/api-reference/types.mdx

        [identifierKey: string]: string
    },
    "credentials": {
        [credentialKey: string]: 

>             string     // Single credential value when `children` is not present and `multi` is false

Author: I didn't phrase it like "set to `true/false`" here because I wanted to shorten the comments.

>             string     // Single credential value when `children` is not present and `multi` is false

Reviewer: ```suggestion
            string     // Single string value when `multi` is false and `children` isn't included in the credential requests
```

>             | [string] // Array of credential values when `multi` is true, but no `children`

Reviewer: ```suggestion
            | [string] // Array of string values when `multi` is true and `children` isn't included in the credential requests
```

>             | Object   // Single credential value of type `Object` when `children` is present and `multi` is false

Reviewer: ```suggestion
            | Object   // Single object value when `multi` is false and `children` is included in the credential requests
```

>             | [Object] // Array of credential values of type `Object` when `multi` is true and `children` is present

Reviewer: ```suggestion
            | [Object] // Array of object values when `multi` is true and `children` is included in the credential requests
```
